Description: fixes installation path of systemdialogs example.
 This bug has been reported upstream as
 https://bugreports.qt-project.org/browse/QTBUG-32359
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Bug: https://bugreports.qt-project.org/browse/QTBUG-32359
Last-Update: 2013-07-10

--- a/examples/quick/dialogs/systemdialogs/systemdialogs.pro
+++ b/examples/quick/dialogs/systemdialogs/systemdialogs.pro
@@ -13,5 +13,5 @@ EXAMPLE_FILES = \
     FileDialogs.qml \
     ColorDialogs.qml
 
-target.path = $$[QT_INSTALL_EXAMPLES]/quick/dialogs
+target.path = $$[QT_INSTALL_EXAMPLES]/quick/dialog/systemdialogs
 INSTALLS += target
